src/WpfMath/TexFormulaParser.cs

```diff
@@ -550,12 +552,23 @@ private Atom AttachScripts(TexFormula formula, string value, ref int position, A
                 return atom;
 
             // Check whether to return Big Operator or Scripts.
+            var subscriptAtom = subscriptFormula == null ? null : subscriptFormula.RootAtom;
+            var superscriptAtom = superscriptFormula == null ? null : superscriptFormula.RootAtom;

```

Comment:
Looks like a good place to use C# 6 syntax: `subscriptFormula?.RootAtom`, `superscriptFormula?.RootAtom`.